mautic/mautic
# DPMMA-2290 Edit contact group points

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ N ]
| New feature/enhancement? (use the a.x branch)      | [ Y ]
| Deprecations?                          | [ N ]
| BC breaks? (use the c.x branch)        | [ N ]
| Automated tests included?              | [ Y ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR adds the possibility to edit the group points assigned to a Contact.
![image](https://github.com/mautic/mautic/assets/8580942/89c9b15c-a70b-4407-aee4-233210d6cbfa)
![Screenshot from 2024-01-04 13-09-00](https://github.com/mautic/mautic/assets/8580942/d586fcb9-ae00-468e-925b-f261f5746d2f)



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Add some group points (Points -> Manage Groups)
3. Set contact group points (Contact view -> Button Bar Dropdown -> Group points)
4. Check if the points are assigned correctly and displayed in contact view page
5. Check if point triggers work correctly when points are edited
